launch.go

```diff
@@ -80,9 +80,9 @@ func getAccountURL(r *http.Request) (string) {
 func redirectURL(w http.ResponseWriter, r *http.Request) {
 	hostURL := settings.Get("SURVEY_RUNNER_URL")
 
-	token, tokenErr := authentication.GenerateTokenFromPost(r.PostForm)
-	if tokenErr != nil {
-		http.Error(w, fmt.Sprintf("GenerateTokenFromPost failed err: %v", tokenErr), 500)
+	token, err := authentication.GenerateTokenFromPost(r.PostForm)
+	if err != "" {
```

Comment:
This breaks convention everywhere else of checking if err != nil
Any reason why error would be returning an empty string if there wasn't an error?

Reply:
in GoLang you cant have a `nil` string